app/serializers/conditional_logic_form/skip_rule_serializer.rb

```diff
@@ -1,32 +1,5 @@
 # frozen_string_literal: true
 
-# rubocop:disable Metrics/LineLength
-# == Schema Information
-#
-# Table name: skip_rules
-#
-#  id             :uuid             not null, primary key
-#  destination    :string           not null
-#  rank           :integer          not null
-#  skip_if        :string           not null
-#  created_at     :datetime         not null
-#  updated_at     :datetime         not null
-#  dest_item_id   :uuid
-#  mission_id     :uuid
-#  source_item_id :uuid             not null
-#
-# Indexes
-#
-#  index_skip_rules_on_dest_item_id    (dest_item_id)
-#  index_skip_rules_on_source_item_id  (source_item_id)
-#
-# Foreign Keys
-#
-#  fk_rails_...  (dest_item_id => form_items.id)
-#  fk_rails_...  (source_item_id => form_items.id)
-#
-# rubocop:enable Metrics/LineLength
-
```

Comment:
I removed these because they got added when this file was in the root serializers directory but it's not there anymore so they're not getting maintained anymore.